test/Cronofy.Test/CronofyOAuthClientTests/RealTimeScheduling.cs

```diff
@@ -164,7 +172,7 @@ public void CanGetRTSUrlWithCallbackUrlAndCompletedUrl()
                             "\"tzid\":\"Etc/UTC\"," +
                             "\"callback_url\":\"{12}\"" +
```

Comment:
I don't think we should be sending `callback_url` in this form at all any more. It's equivalent to `callback_urls.completed_url`, so the built request should never need to include both.

It's good that the Builder still supports the old singular `CallbackUrl()` method, but it should map to the same underlying value as the first argument of `CallbackUrls()`. i.e. `CallbackUrl("foo")` should be equivalent in behaviour to `CallbackUrls("foo", null, null)`